lib/CompilerImpl/ClangLibCompiler.cpp

```diff
@@ -142,8 +142,7 @@ ClangLibCompiler::generateIR(const std::vector<std::filesystem::path> &src,
     log_str = log_str + arg + " ";
   }
   Compiler::log_string(log_str);
-
-  driver::Driver NikiLauda(_llvmManager->getClangExePath().string(),
+  clang::driver::Driver NikiLauda(_llvmManager->getClangExePath().string(),
```

Comment:
Not sure why we suddenly need the `clang` namespace.
Is it for every version?

Reply:
this was something that my ide was showing as an issue, I will try to see if it works without an update as required

Reply:
I checked an as both clang and llvm namespaces have a ::driver candidate, there is an ambiguity that causes the compilation to fail, as such I think it is required. Definitely need your input on this

Reply:
We could add a workaround, like `using` clause, but I think that your disambiguation works fine.